PennyPincher/PennyPincherAndroidGesturesImporter.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  PennyPincherAndroidGesturesImporter.swift
+//  PennyPincher
+//
+//  Created by Raf Cabezas on 1/30/17.
+//  Copyright © 2017 WillowTree, Inc. All rights reserved.
+//
+
+import Foundation
+
+// Example usage:
+//
+//    for gesture in PennyPincherAndroidGesturesImporter.translatedGestures(from: PennyPincherAndroidGesturesImporter.defaultAndroidFilePath ?? "") {
+//        if let template = PennyPincher.createTemplate(gesture.id, points: gesture.allPoints) {
+//            pennyPincherGestureRecognizer.templates.append(template)
+//        }
+//    }
+//
+
+public struct ImportedStrokes {
+    public let points: [CGPoint]
+    
+    public init(points: [CGPoint]) {
+        self.points = points
+    }
+}
+
+public struct ImportedGesture {
```

Comment:
Conceivably you could import gestures from other formats than Android's GestureStore. Are these types generic enough to work for all such places? If so, they deserve their own files. If not, they deserve to mention Android GestureStore by name (if that is indeed the best name) … or they could be inner types of the importer type.